testproject/Assets/Tests/Runtime/RpcTestsAutomated.cs

```diff
@@ -0,0 +1,164 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using TestProject.ManualTests;
+using MLAPI.RuntimeTests;
+using MLAPI;
+using Debug = UnityEngine.Debug;
+
+namespace TestProject.RuntimeTests
+{
+    public class RPCTestsAutomated
+    {
+        private bool m_TimedOut;
+
+        /// <summary>
+        /// Default Mode (Batched RPCs Enabled)
+        /// 1 Host and 9 Clients
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManualRpcTestsAutomated()
+        {
+            return AutomatedRpcTestsHandler(9);
+        }
+
+        /// <summary>
+        /// Same test with Batched RPC turned off
+        /// 1 Host and 3 Clients
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManualRpcTestsAutomatedNoBatching()
+        {
+            return AutomatedRpcTestsHandler(3, false);
+        }
+
+
+        /// <summary>
+        /// This just helps to simplify any further tests that can leverage from
+        /// the RpcQueueManualTests' wide array of RPC testing under different
+        /// conditions.  
+        /// Currently this allows for the adjustment of client count and whether
+        /// RPC Batching is enabled or not.
+        /// </summary>
+        /// <param name="numClients"></param>
+        /// <param name="useBatching"></param>
+        /// <returns></returns>
+        private IEnumerator AutomatedRpcTestsHandler(int numClients, bool useBatching = true)
+        {
+            // Set RpcQueueManualTests into unit testing mode
+            RpcQueueManualTests.UnitTesting = true;
+
+            // Create Host and (numClients) clients 
+            Assert.True(MultiInstanceHelpers.Create(numClients, out NetworkManager server, out NetworkManager[] clients));
+
+            // Create a default player GameObject to use
+            var playerPrefab = new GameObject("Player");
+            var networkObject = playerPrefab.AddComponent<NetworkObject>();
+
+            // Add our RpcQueueManualTests component
+            playerPrefab.AddComponent<RpcQueueManualTests>();
+
+            // Make it a prefab
+            MultiInstanceHelpers.MakeNetworkedObjectTestPrefab(networkObject);
```

Comment:
I think the name of this API should change.
```suggestion
            MultiInstanceHelpers.MakeNetworkObjectTestPrefab(networkObject);
```

Reply:
This code...the method you are pointing to is not new or changed nor part of this PR.
Perhaps open a new PR and make changes to MultiInstanceHelpers?
Changing the name of a method for and already existing class is sort of outside of the scope of this specific PR.

Reply:
ugh, okay

Reply:
Unless @TwoTenPvP decides to make this change?
He will be doing a final pass on this PR to make any additional tweaks to some of the changes in the MultiInstanceHelpers and SIPTansport.